Source/CarthageKit/Xcode.swift

```diff
@@ -630,26 +630,23 @@ private func build(sdk: SDK, with buildArgs: BuildArguments, in workingDirectory
 		// Specifying destination seems to be required for building with
 		// simulator SDKs since Xcode 7.2.
 		if sdk.isSimulator {
-			let destinationLookup = Task("/usr/bin/xcrun", arguments: [ "simctl", "list", "devices" ])
+			let destinationLookup = Task("/usr/bin/xcrun", arguments: [ "simctl", "list", "devices", "--json" ])
 			return destinationLookup.launch()
 				.ignoreTaskData()
 				.map { data in
-					let string = String(data: data, encoding: .utf8)!
-					// The output as of Xcode 6.4 is structured text so we
-					// parse it using regex. The destination will be omitted
-					// altogether if parsing fails. Xcode 7.0 beta 4 added a
-					// JSON output option as `xcrun simctl list devices --json`
-					// so this can be switched once 7.0 becomes a requirement.
```

Comment:
I think it doesn't need to support Xcode 6.x yet. So I dropped regexp version.
How do you think?